common/schema/DHQauthor-TEI.xml

```diff
@@ -268,13 +271,14 @@ email (should only have an email address)
         <!-- Modules for DHQ: TEI modules, the DHQ module, and MathML (in that order) -->
 
         <moduleRef key="core"
-          except="abbr add addrLine analytic biblStruct binaryObject cb choice corr del distinct divGen expan gap gloss headItem headLabel imprint index measure measureGrp mentioned milestone monogr num orig pb postBox postCode reg relatedItem resp respStmt rs sic street teiCorpus time unclear"/>
+          except="abbr add addrLine binaryObject cb choice corr del distinct divGen expan gap gloss headItem headLabel index measure measureGrp mentioned milestone num orig pb postBox postCode reg relatedItem resp respStmt rs sic street teiCorpus time unclear"/>
         <moduleRef key="tei"/>
         <moduleRef key="header"
           except="abstract appInfo application authority biblFull catDesc catRef category classCode correction distributor edition editionStmt editorialDecl extent funder geoDecl handNote hyphenation interpretation namespace normalization notesStmt principal projectDesc quotation refState refsDecl rendition samplingDecl segmentation seriesStmt stdVals styleDefDecl tagUsage tagsDecl"/>
         <moduleRef key="textstructure"
           except="argument byline closer div1 div2 div3 div4 div5 div6 div7 docAuthor docDate docEdition docImprint docTitle imprimatur opener titlePage titlePart"/>
         <moduleRef key="figures"/>
+      	<moduleRef key="namesdates" include="persName orgName placeName surname forename"/>
```

Comment:
Obviously necessary (given that our `<biblStruct>`s use `<persName>` with `<surname>` and `<forename>` children). But I suspect you don’t want these elements showing anywhere _except_ inside `<biblStruct>`. If so, we need to either

1. Remove these elements from their common classes (e.g., `<persName>` from model.nameLike.agent) and add them back to content model of places the are actually needed (e.g., `<author>`); OR
2. Add a Schematron constraint that whines if one of these is outside a `<biblStruct>`.

Reply:
(Does not have to be done on _this_ PR, of course.)